engine/classes/ElggEntity.php

```diff
@@ -1323,7 +1321,7 @@ public function removeFromSite($site) {
 	/**
 	 * Gets the sites this entity is a member of
 	 *
-	 * Site membership is determined by relationships and not site_guid.
+	 * Site membership is determined by relationship.
```

Comment:
relationship does not have to be created at the same time as the entity... so it makes sense... could be discussed in another ticket